### DISCUSSION TITLE
Welcome to Shairport Sync Discussions!

### DISCUSSION THREAD

**Discussion by mikebrady** [mikebrady/shairport-sync#1219] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by charlesomer** [comment]
Hello everyone 👋 Just want to say another big thank you to @mikebrady for his work on this! I'll be getting the Docker builds up and running again soon as well as the upcoming documentation website 👀

**Reply by bibberp** [reply]
Love the work going on here, shairport-sync has been a huge blessing. Is there a docker build for this or just the bare metal install for ap2 support?

**Reply by charlesomer** [reply]
There’s one that gets built based on the ‘development’ branch: `mikebrady/shairport-sync:unstable-development` 😄

**Reply by bibberp** [reply]
Thank you!

**Reply by bibberp** [reply]
Another question, is it possible to run the container on windows? I've tried before but was unsuccessful (linux subsystem)

**Reply by mikebrady** [reply]
Interesting question, but I'm afraid I don't know. Shairport Sync relies on access to the `alsa` subsystem, and I don't know if that's implemented in the linux subsystem...
